### PR TITLE
Added the Pleistocene version of the CPT-Vs correlation from Andrus et al. (2007)

### PR DESCRIPTION
**Pull request by AndrewRidden-Harper**
Jaehwi found that the Pleistocene version of the CPT-Vs correlation from Andrus et al. (2007) performs the best but it wasn't available in vs_calc. I added the Pleistocene version, and also noticed a version for "tertiary age cooper marl" so I added that as well. I also added the Andrus et al. (2007) reference paper to the repo, with the idea that all reference papers with correlations used in vs_calc could also be added in the future.

### CODE REVIEW

File: VsViewer/vs_calc/cpt_vs_correlations.py [hunk 1]
```diff
@@ -45,18 +45,55 @@ def mcgann_2018(cpt: CPT):
     return VsMcGann2, Vs_SD
 
 
-def andrus_2007(cpt: CPT):
+def andrus_2007_holocene(cpt: CPT):
     """
-    CPT-Vs correlation developed by Andrus et al. (2007).
-    qt in kPa
+    CPT-Vs correlation developed by Andrus et al. (2007) for Holocene-age soils (Equation 5).
+    qt in kPa.
```
Comment (lispandfound) [inline VsViewer/vs_calc/cpt_vs_correlations.py:51]: Could you run copilot over this file and get it to produce numpy-style docstrings for the functions in this module?

File: VsViewer/vs_calc/cpt_vs_correlations.py [hunk 2]
```diff
@@ -45,18 +45,55 @@ def mcgann_2018(cpt: CPT):
...
+
+
+def andrus_2007_tertiary_age_cooper_marl(cpt: CPT):
+    """
+    CPT-Vs correlation developed by Andrus et al. (2007) for Tertiary-Age Cooper Marl deposits (Equation 9).
+    qt in kPa.
+    """
+    cpt.qt[cpt.qt <= 0] = 0.0001  # adjust for possible negative qt
+    VsAnd = np.array(
+        [13.0 * ((cpt.qt * 1000) ** 0.382) * (cpt.depth**0.099)]
```
Comment (lispandfound) [inline VsViewer/vs_calc/cpt_vs_correlations.py:92]: The formatting here is not good, could you also run ruff over the file to correct the formatting?

**Review by lispandfound — CHANGES_REQUESTED** [review]
Just formatting changes here.

**Review by joelridden — APPROVED** [review]
Nice, formatting would be nice, eventually should get ruff etc for this repo to tidy this all up

**Review by lispandfound — APPROVED** [review]
approve on the proviso of ruff formatting the repo

**Pull request merged**